docker-compose.yml

```
@@ -105,13 +95,14 @@ services:
       KEYCLOAK_PASSWORD: Pa55w0rd
       KEYCLOAK_IMPORT: /opt/jboss/keycloak/imports/realm-export.json -Dkeycloak.profile.feature.upload_scripts=enabled
       PROXY_ADDRESS_FORWARDING: 'true'
+      KEYCLOAK_FRONTEND_URL: 'https://auth.brokencrystals.neuralegion.com/auth/'
     healthcheck:
       test:
         [
           'CMD',
           'curl',
           '-f',
-          'http://localhost:8080/auth/realms/brokencrystals/health/check/database',
+          'https://auth.brokencrystals.neuralegion.com/auth/realms/master',
```

You have to use the internal address 

```suggestion
          'http://localhost:8080/auth/realms/master',
```